diff -Naur sabctools-9.4.0.old/pyproject.toml sabctools-9.4.0/pyproject.toml
--- sabctools-9.4.0.old/pyproject.toml	2026-05-10 10:22:39.697398393 +0200
+++ sabctools-9.4.0/pyproject.toml	2026-05-10 10:41:29.287480057 +0200
@@ -11,8 +11,7 @@
     {name = "Safihre", email = "safihre@sabnzbd.org"},
 ]
 requires-python = ">=3.9"
-license = "GPL-2.0-or-later"
-license-files = ["LICENSE.md"]
+license = {text = "GPL-2.0-or-later"}
 classifiers = [
     "Programming Language :: Python",
     "Programming Language :: Python :: 3 :: Only",
